packages/custom-elements-manifest-tools/src/lib/get-custom-elements.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  CustomElementDeclaration,
+  CustomElementExport,
+  Module,
+  Package,
+} from 'custom-elements-manifest/schema';
+import {resolveReference} from './resolve-reference.js';
+
+export type CustomElementInfo = {
+  package: Package;
+  module: Module;
+  export: CustomElementExport;
+  declaration: CustomElementDeclaration;
+};
+
+/**
+ * Gets all the custom element exports of a package
+ */
+export const getCustomElements = (
```

Comment:
Why always use arrow functions instead of `function` keyword? Pro: lexical `this` (though not usually relevant in these module functions), Con: name isn't shown in stack trace (or is it?).

Reply:
The name should be shown - it's taken from the variable name.

I like it for consistency, not allowing access to `arguments`, the `this` handling, and so that it's not a mutable export. And I basically avoid `function` outside of generators now. Very minor differences though.